tests/e2e/specs/edit-posts.test.js

```diff
@@ -1,29 +1,32 @@
-import {
-	createNewPost,
-	pressKeyTimes,
-	publishPost,
-	trashAllPosts,
-	visitAdminPage,
-} from '@wordpress/e2e-test-utils';
-
-describe( 'Edit Posts', () => {
-	beforeEach( async () => {
-		await trashAllPosts();
+/**
+ * WordPress dependencies
+ */
+import { test, expect } from '@wordpress/e2e-test-utils-playwright';
+
+test.describe( 'Edit Posts', () => {
+	test.beforeEach( async ( { requestUtils }) => {
+		await requestUtils.deleteAllPosts();
 	} );
 
-	it( 'displays a message in the posts table when no posts are present', async () => {
-		await visitAdminPage( '/edit.php' );
-		const noPostsMessage = await page.$x(
+	test( 'displays a message in the posts table when no posts are present',async ( {
+		admin,
+		page,
+	} ) => {
+		await admin.visitAdminPage( '/edit.php' );
+		await expect( page.locator(
 			'//td[text()="No posts found."]'
```

Comment:
This can be replaced with a role-based selector `getByRole( 'cell', { name: 'No posts found.' } )`

Reply:
This is an odd one. Chrome a11y tree gives the element the role of `gridcell`, but PW sees it as a `cell`.

Reply:
Maybe a different browser version or something?

Reply:
FYI Playwright doesn't use the chromium accessibility API for `getByRole`. They implement their own resolution logic (following the standard ofc) in JS instead to solve the cross-browser compatibility issue. So in short the devtools in chromium isn't always going to match what Playwright sees 😆.